### PR TITLE
Fixed user profile access at blanks page (Fixed issues #2195 #2194 #2196)

### PR DESCRIPTION
**Pull request by petromonka**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by sharix4 — CHANGES_REQUESTED** [review]
TESTS

**Review by IlliaDR — APPROVED** [review]

**Review by 0l9h — APPROVED** [review]

File: EPlast/EPlast.BLL/Interfaces/UserProfiles/IUserService.cs [hunk 1]
```diff
@@ -90,6 +90,15 @@ public interface IUserService
...
         Task<string> GetUserGenderAsync(string userId);
 
+        /// <summary>
+        /// Function checks if users are in same city, region or hovel
+        /// </summary>
+        /// <param name="currentUser">active user</param>
+        /// <param name="focusUser">user which we checking</param>
+        /// <param name="cellType">Type of cell (City, Region , Hovel)</param>
+        /// <returns></returns>
+        Task<bool> IsUserInSameCell(UserDTO currentUser, UserDTO focusUser, CellType cellType);
```
Comment (OleksandrBilan) [inline EPlast/EPlast.BLL/Interfaces/UserProfiles/IUserService.cs:100]: IsUserInSameCellAsync*

File: EPlast/EPlast.BLL/Interfaces/UserProfiles/IUserService.cs [hunk 2]
```diff
@@ -90,6 +90,15 @@ public interface IUserService
         /// <returns>User gender string</returns>
         Task<string> GetUserGenderAsync(string userId);
 
+        /// <summary>
+        /// Function checks if users are in same city, region or hovel
+        /// </summary>
+        /// <param name="currentUser">active user</param>
+        /// <param name="focusUser">user which we checking</param>
+        /// <param name="cellType">Type of cell (City, Region , Hovel)</param>
+        /// <returns></returns>
```
Comment (OleksandrBilan) [inline EPlast/EPlast.BLL/Interfaces/UserProfiles/IUserService.cs:99]: remove useless line

**Review by danylo-melnychuk — APPROVED** [review]

File: EPlast/EPlast.BLL/Interfaces/UserProfiles/IUserService.cs (hunk 1 above)
Comment (petromonka) [reply EPlast/EPlast.BLL/Interfaces/UserProfiles/IUserService.cs]: I always forget to add this suffix. Thank you for remind 👍

**Comment by petromonka** [comment]
> TESTS

Tests✅ are added, so please approve this PR... 
Thank you in advance 👍

**Review by sharix4 — APPROVED** [review]

**Pull request merged**